docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2952,11 +2954,11 @@ Create a volume
       "Name": "tardis",
       "Driver": "local",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
```

Comment:
Yes; basically the same, but for *responses*; for example, you cannot set `Status`, or `Scope` when *creating* a volume, but the *response* from the daemon may include a `Status` or `Scope` field. We currently don't document those fields anywhere (I think).

Adding a `**JSON fields**:` section to describe the fields that are *returned* by the daemon would solve that.

However, I don't think that change needs to be part of this PR per se, just think it would be a good improvement to the docs.